pom.xml

```
@@ -112,7 +112,7 @@
         <dependency.okhttp3.version>4.7.2</dependency.okhttp3.version>
         <dependency.gson.version>2.8.8</dependency.gson.version>
 
-        <plugin.surefire.version>2.22.2</plugin.surefire.version>
+        <plugin.surefire.version>3.0.0-M3</plugin.surefire.version>
```

Please use latest milestone version:

```suggestion
        <plugin.surefire.version>3.0.0-M5</plugin.surefire.version>
```